script.js

```diff
@@ -0,0 +1,73 @@
+const toastSandbox = document.querySelector('#toast-sandbox');
+function updateStyles(selector) {
+  if (toastSandbox !== document.querySelector(selector))
+    toastSandbox.textContent = document.querySelector(selector).textContent;
+  document.querySelector('#toast-styler').innerHTML = toastSandbox.value;
+};
+updateStyles('#toast-sandbox');
+
+toastSandbox.addEventListener('input', () => {updateStyles('#toast-sandbox');});
+
+toastSandbox.addEventListener('input', () => {console.log(toastSandbox.textContent)});
+
+// Resize textareas
+const updateTextareaSize = (area) => {
+  const numLines = area.value.split('\n').length;
+  area.style.minHeight = `${numLines*(1.15)}em`;
+}
+
+const textareaList = document.querySelectorAll('textarea');
+textareaList.forEach((area) => {
+  updateTextareaSize(area);
+  area.addEventListener('input', () => {updateTextareaSize(area);})
+});
+
+// DEMO CUSTOM ELEMENT
+const stylesheet = new CSSStyleSheet();
+      stylesheet.replaceSync(`
+        .code-area {
```

Comment:
Not clear why this styling is in the custom element instead of in styles.css. You could do `::slotted(css)` and remove the `class="code-area"` everywhere, if you keep it here.

In fact, it seems this is duplicated in styles.css?

Reply:
This was actually fine to just remove, something I missed when switching things to textareas